bin/echo.c

```diff
@@ -93,7 +93,7 @@ int early_data_recv(struct s2n_connection *conn)
 
     if (total_data_recv > 0) {
         fprintf(stdout, "Early Data received: ");
-        for (size_t i = 0; i < total_data_recv; i++) {
+        for (size_t i = 0; i < (size_t)total_data_recv; i++) {
```

Comment:
Why not make i a ssize_t? This loop is only iterating up to total_data_recv which is a ssize_t so it can't go past that number of rounds.